schema.graphql

```diff
@@ -50,6 +50,7 @@ input AddMultipleFileStatusesInput {
 input AddOrUpdateBulkFileMetadataInput {
   consignmentId: UUID!
   fileMetadata: [AddOrUpdateFileMetadata!]!
+  skipValidation: Boolean = false
```

Comment:
I wouldn't allow this input directly. Need to ensure that clients don't skip the validation when they shouldn't.

I'd restructure the approach.

Change the signature of the method: `addOrUpdateBulkFileMetadata` to take the access token:

> `addOrUpdateBulkFileMetadata(metadataInput: AddOrUpdateBulkFileMetadataInput, token: Token)`

Change to signature of the `addFileMetadataStatuses` to take the token:

> `addFileMetadataStatuses(metadataInput: AddOrUpdateBulkFileMetadataInput, token: Token)`

Use the token to check for the new draft metadata client role `update_metadata` to determine whether to skip the validation or not.

Reply:
Added a check for a user to not allow to skip the validation. Thanks